DS13/code/modules/EMH/EMH.dm

```diff
@@ -0,0 +1,276 @@
+GLOBAL_VAR_INIT(EMH_present, FALSE) //Stop multiple holograms for now...To avoid mass grief.
+GLOBAL_LIST_INIT(EMH_blacklist, list())
+
+//EMITTER
+
+/obj/machinery/emh_emitter
+	name = "Emergency medical hologram emitter"
+	desc = "Simply click this device and an emergency medical hologram will be summoned. Swipe it with a captain level ID to terminate the current EMH if it's misbehaving."
+	icon = 'DS13/icons/obj/decor/wall_decor.dmi'
+	icon_state = "emh-off"
+	var/mob/living/carbon/human/species/holographic/emh
+	anchored = TRUE
+	density = FALSE
+	pixel_y = 32 //Put on the tile below a wall etc etc modular fun time mapping mayhem
+	var/range = 10 //10 tiles? Pretty damn generous.
+	req_access = list(ACCESS_CAPTAIN)
+
+/obj/machinery/emh_emitter/examine(mob/user)
+	. = ..()
+	if(GLOB.EMH_present)
+		if(emh.client)
+			to_chat(user, "<i>The panel shows that there is an active EMH on the network.</i>")
+		else
+			to_chat(user, "<i>The panel shows that there is an EMH on the network, but it is not responding (braindead). Click the panel to summon a new one.</i>")
+
+/mob/verb/editemhblacklist()
+	set name = "Edit EMH blacklist"
+	set desc = "Remove people from the EMH blacklist for this round."
+	set category = "Admin"
+	if(!check_rights(R_ADMIN, TRUE)) //We may want to change this to R_VAREDIT when we launch but for now I'd quite like mentors to have this too.
+		return
+	var/A
+	var/list/temp = GLOB.EMH_blacklist.Copy()
+	temp += "cancel" //To stop the input only having one option, which makes byond freak out.
+	A = input("Remove whom from the blacklist?", "Admin tools", A) as null|anything in GLOB.EMH_blacklist
+	if(!A)
+		return
+	if(A in GLOB.EMH_blacklist)
+		GLOB.EMH_blacklist -= A
+	log_admin("([worldtime2text()]):[src] / [client.ckey] removed [A] from the EMH blacklist for this round.")
+
+/obj/machinery/emh_emitter/attackby(obj/I,mob/user)
+	. = ..()
+	if(user.client)
+		var/obj/item/card/id/ID = I
+		if(ID && istype(ID))
+			if(check_access(ID))
+				var/question = alert("Terminate the current EMH? (this kills the current EMH)",name,"yes","no")
+				if(question == "yes")
+					if(emh.client && emh.client.key)
+						if(!emh.client.key in GLOB.EMH_blacklist)
+							to_chat(user, "Blacklist the current EMH? (THIS MEANS THEY CANNOT BECOME AN EMH AGAIN THIS ROUND. ADMINS WILL BE NOTIFIED OF THIS, SO ONLY DO THIS IF THE EMH IS GRIEFING / BEING A MAJOR INCONVENIENCE! FALSE BLACKLISTING MAY LEAD TO A BAN!")
```

Comment:
`policy` and also extreme screaming.

Reply:
Remove the ahelp alert at L61 as well

Reply:
This should be set as a thing at config. Before this PR is merged either we leave this as is and do it as a issue report to resolve or move that to a config with text inside that config for what is said here. I believe that is the proper practice.